### PR TITLE
CyberArk - Integration rename and consolidation

### PR DESCRIPTION
**Pull request by nitsan-tzur**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by nitsan-tzur** [comment]
> Hi @nitsan-tzur, due to Github technical issue, can you please re-sign the [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=37362)? This should help `license/cla` Github action to run.

Hi @samuelFain SLA signed

File: Packs/CyberArkEPV/README.md [hunk 1]
```diff
@@ -1,47 +1,34 @@
...
+# CyberArk PAM Self-Hosted
 
 <~XSIAM>
 
 This pack includes Cortex XSIAM content.
 
 ## Configuration on Server Side
 
-This section describes the steps required to configure Syslog forwarding of vault audit logs, such as user activity and safe activity events, from CyberArk EPV to Cortex XSIAM.
+This section describes the steps required to configure Syslog forwarding of vault audit logs, such as user activity and safe activity events, from CyberArk PAM Self-Hosted Vault to Cortex XSIAM.
```
Comment (ShirleyDenkberg) [inline Packs/CyberArkEPV/README.md:10]: Is it **PAM Self-Hosted** or **PAM Self-Hosted Vault**?

File: Packs/CyberArkEPV/README.md [hunk 2]
```diff
@@ -1,47 +1,34 @@
...
-     </xsl:call-template> 
-      ```
-      See the following screenshot for an example of the updated XSL file: 
-    ![xsl_with_timestamp_mapping](https://raw.githubusercontent.com/demisto/content/fcf4535d373df78bded4b1bedacdd505d25cc095/Packs/CyberArkEPV/doc_files/config_isotimestamp_mapping.png)
-4. Save the changes.
+the XML event records must be converted to suitable [CEF](https://www.microfocus.com/documentation/arcsight/arcsight-smartconnectors-8.3/cef-implementation-standard/Content/CEF/Chapter%201%20What%20is%20CEF.htm) messages. 
+
+### Set up the XSL Translator
+This transformation from XML event records to CEF messages is done though a suitable [XSL](https://en.wikipedia.org/wiki/XSL) translator file. 
+Example of such XSL file can be found [here](https://raw.githubusercontent.com/demisto/content/fcf4535d373df78bded4b1bedacdd505d25cc095/Packs/CyberArkEPV/doc_files/XSIAM.xsl). This file can be used directly within the target vault.
```
Comment (ShirleyDenkberg) [inline Packs/CyberArkEPV/README.md:19]:
```suggestion
An example of an XSL file can be found [here](https://raw.githubusercontent.com/demisto/content/fcf4535d373df78bded4b1bedacdd505d25cc095/Packs/CyberArkEPV/doc_files/XSIAM.xsl). This file can be used directly within the target vault.
```

File: Packs/CyberArkEPV/ReleaseNotes/1_0_4.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+#### Parsing Rules
+
+##### CyberArk EPV Parsing Rule
+
+Add Vault Translator XSL file to be downloaded from pack and used directly. 
+
+Change naming from Enterprise Password Vault (EPV) to PAM-Self Hosted
```
Comment (ShirleyDenkberg) [inline Packs/CyberArkEPV/ReleaseNotes/1_0_4.md:8]:
```suggestion
- Added Vault Translator XSL file to be downloaded from pack and used directly. 
- Changed naming from Enterprise Password Vault (EPV) to PAM-Self Hosted
```

**Comment by ShirleyDenkberg** [comment]
@MLainer1 Doc review completed.

**Comment by MLainer1** [comment]
Hi @nitsan-tzur, before we merge your PR, can you look at Shirley's Doc review? 
Afterwards, we will merge the PR.
Thanks

**Review by MLainer1 — APPROVED** [review]

**Pull request merged**